packages/cwd-voting/src/multiple_choice.rs

```diff
@@ -139,24 +166,39 @@ impl MultipleChoiceOptions {
     }
 }
 
-// we cast a ballot with our chosen vote and a given weight
-// stored under the key that voted
-#[derive(Serialize, Deserialize, Clone, PartialEq, Eq, JsonSchema, Debug)]
-pub struct Ballot {
-    /// The amount of voting power behind the vote.
-    pub power: Uint128,
-    /// The position.
-    pub vote: MultipleChoiceVote,
-}
+#[cfg(test)]
+mod test {
+    use super::*;
 
-pub const CONFIG: Item<Config> = Item::new("config");
-pub const PROPOSAL_COUNT: Item<u64> = Item::new("proposal_count");
-pub const PROPOSALS: Map<u64, MultipleChoiceProposal> = Map::new("proposals");
-pub const BALLOTS: Map<(u64, Addr), Ballot> = Map::new("ballots");
-pub const PROPOSAL_HOOKS: Hooks = Hooks::new("proposal_hooks");
-pub const VOTE_HOOKS: Hooks = Hooks::new("vote_hooks");
+    #[test]
+    fn test_display_multiple_choice_vote() {
+        let vote = MultipleChoiceVote { option_id: 0 };
+        assert_eq!("0", format!("{}", vote))
```

Comment:
i think this test is actually circular (always true) as the format call will also use the display implementation :)